src/libraries/System.Net.Quic/src/System/Net/Quic/QuicException.cs

```diff
@@ -75,4 +75,8 @@ internal QuicException(QuicError error, long? applicationErrorCode, long? transp
     /// The transport protocol error code associated with the error.
     /// </summary>
     public long? TransportErrorCode { get; }
+
```

Comment:
Please don't merge this particular change, that's just for debugging from where the UOE comes from :smile: